src/components/Button/buttonRules.ts

```diff
@@ -91,12 +98,15 @@ export default {
 
       ...(secondary && {
         color: typeSecondaryColor,
-        backgroundColor: typeSecondaryBackgroundColor,
+        backgroundColor: active
+          ? secondaryActiveStyle.backgroundColor
+          : typeSecondaryBackgroundColor,
         borderColor: typeSecondaryBorderColor,
         ':hover': {
           borderColor: 'transparent',
           backgroundColor: typeSecondaryBackgroundColorHover,
         },
+        borderWidth: `${!active ? (circular ? pxToRem(1) : pxToRem(2)) : 0}`,
```

Comment:
the same here - more succinct expression could be used, to avoid duplication

Reply:
nit: also, it would be a bit easier to read if there would be no negation in the condition:
```js
 borderWidth: `${active ? 0 : pxToRem(circular ? 1 : 2) }`,
```